apps/app_chanisavail.c

```diff
@@ -175,9 +180,11 @@ static int chanavail_exec(struct ast_channel *chan, const char *data)
 			status = inuse = ast_device_state(trychan);
 		}
 		ast_str_append(&tmp_availstat, 0, "%s%d",
-			ast_str_strlen(tmp_availstat) ? "&" : "", status);
+		ast_str_strlen(tmp_availstat) ? "&" : "", status);
```

Comment:
Indentation is off here now

Reply:
It looked broken before (like the indentation was incorrect) so I just updated it since I was already touching the code around here.  I can revert it though if you'd like.

Reply:
Oh, I see it now.  That second function was an arg to the first one.  I'm going to try to just make that one line if you're ok with it so that it's not confusing as it was before.